File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        uniqueNums = set(nums)

        longestCons = 0
        for num in uniqueNums:
            if num - 1 not in uniqueNums:
                counter = 0
                while num in uniqueNums: 
                    counter += 1
                    num += 1
                longestCons = max(counter, longestCons)

        return longestCons

File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        int[] copyArray = new int[nums.length * 2]; 
        int[] finalArray = new int [nums.length + copyArray.length];

        for (int i = 0; i < copyArray.length; i++)
        {
            if (i < nums.length)
            {
                copyArray[i] = nums[i];
            }
            else {
                copyArray[i] = nums[i - nums.length];
            }

        }

        return copyArray; 
    }
}
